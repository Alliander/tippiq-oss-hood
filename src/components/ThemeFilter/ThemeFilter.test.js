import React from 'react';
import renderer from 'react-test-renderer';
import Element from './index';

test('Renders', () => {
  const component = renderer.create(
    <Element />
  );
  const json = component.toJSON();
  expect(json).toMatchSnapshot();
});

