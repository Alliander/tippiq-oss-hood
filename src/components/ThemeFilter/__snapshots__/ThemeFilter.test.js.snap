exports[`test Renders 1`] = `
<div
  className="theme-filter-container">
  <h2>
    Thema\'s
  </h2>
  <ul
    className="themes-filter" />
</div>
`;
